src/Tokenomia/Vesting/PrivateSale.hs

```diff
@@ -0,0 +1,235 @@
+{-# LANGUAGE BlockArguments #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE ImportQualifiedPost #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeOperators #-}
+
+module Tokenomia.Vesting.PrivateSale (verifyPrivateSale) where
+
+import Ledger qualified
```

Comment:
This is up to you I guess, as I don't _think_ we have a rule on this, but if you remove the blank lines between each group of imports fourmolu will keep them sorted. Are you trying to keep them organized by Cardano/Plutus, `base`, "other", etc...?

Reply:
Didn't know exactly how to organize them but I also didn't know fourmolu would sort them like that. I have removed the blank lines.